### ISSUE TITLE
Does not work with tmux on sway wayland

### ISSUE THREAD

**Issue by horhenani** [jstkdng/ueberzugpp#218]
terminate called after throwing an instance of 'std::system_error
  what():  Connection refused

[1]  + 102032 IOT instruction (core dumped)  ueberzug layer

I am getting this error using tmux on wayland. 
Command used: ueberzug layer &

**Comment by jstkdng** [comment]
Does it work outside of tmux?

**Comment by horhenani** [comment]
No, it doesn't work. Then I enable xwayland on sway, command "ueberzug layer &" works but it doesn't render any image using ranger or yazi.

**Comment by ovwxxwvo** [comment]
hey, I got something new for this env. 
ueberzugpp does not get the correct window_cell width & height from tmux.
```
tmux display-message -p '#{window_cell_width}'
tmux display-message -p '#{window_cell_height}'
tmux display-message -p '#{window_width}'
tmux display-message -p '#{window_height}'
```
when I set the x of image to the window_width, image go to the wrong position.
if i not set the scaler option, adjust width|height can not change the show of image size. 
if i set the scaler option, the show of image size only can zoom out.
when I set the width & height small than the show of image size , the terminal crash down.

![Uploading 20241112_161133.png…]()

**Comment by ovwxxwvo** [comment]
> Does it work outside of tmux?

I  test this before, in xorg i3 by ueberzug,  ueberzug can get the correct position of tmux,
& after I zoom in|out the alacritty, ueberzug can adjust the image position.
BUT no matter I zoom in|out ueberzugpp, don't change the image position in the term.
by the mean, what is the unit of width & height, is cell ?

**Comment by jstkdng** [comment]
yes, the cells are calculated and then the font size is calculated, but only once, any font changes after the fact are not taken into account

**Comment by ovwxxwvo** [comment]
> yes, the cells are calculated and then the font size is calculated, but only once, any font changes after the fact are not taken into account

ok, thanks, I just giving up on `ueber`.
